File: hogehoge/sura2yomeruPython/chap4/chap4-3-1.py
# メール定型文の受信者に任意の文字列を差し込む
# python3

def create_mail(recv, bill):
# create_mailという名前で、引数reacvと引数billを受けとる以下の関数を作る

    tmp = '''{}様
PT企画の斉藤です。
今月の請求額は{}円です。
'''
# 変数tmpに次の三重クォート文字列を入れろ
# 『{}様
# PT企画の斉藤です。
# 今月の請求額は{}円です。』

    msg = tmp.format(recv, bill)
# 変数tmpに対して、引数recvと引数billを差し込んだ結果を変数msgに入れろ

    print(msg)
# 変数msgを表示しろ

create_mail('山本', 40000)
# 文字列「山本」と数値40000を指定してメールを作れ

File: hogehoge/JavaScript/surasura/1_1/chap4_3_1.js
// メール定型文の受信者名に任意の文字列を差し込む
let createMail = (recv, bill) =>{
    let msg = `${recv}様
PT企画の斉藤です。
今月の請求額は${bill}円です。`;
    console.log(msg);
};
createMail('山本', 40000);

// 引数'recv'と'bill'を受けとる以下の関数を作って変数'createMail'に入れろ。
// { 変数'msg'に、次のテンプレート文字列を入れろ
// 『${recv}様
// PT企画の斉藤です。
// 今月の請求額は${bill}円です。
// 変数'msg'を表示しろ。　}
// 文字列'山本'と、'40000'を指定してメールを作れ。